mos-platform/atari8-stdcart/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+platform(atari8-stdcart COMPLETE PARENT atari8)
+
+if(NOT CMAKE_CROSSCOMPILING)
+  return()
+endif()
+
+install(FILES
+  cart-rom-fixed.ld
+  link.ld
+  stdcart.ld
+TYPE LIB)
+
+# do not create crt0, that will precedence over the common version
+# which we depending on; rather create something specific
+add_platform_library(atari8-stdcart-c
+  syms.s)
+merge_libraries(atari8-stdcart-c
```

Comment:
Similarly, it's not a good idea to run around the atari8 target like this; the policy has been that derived targets add to, but to not alter, their parents.

Accordingly, it'll be a bit of work, but we should really break out the common bits of the `atari8` target into an `atari8-common` target. Then atari8 and atari8-stdcart would be thin descendants that fill out the specifics of that configuration. This is the same relationship that `c64` and `cx16` have to commodore, and the various `nes` targets have to `nes`.